jdaviz/configs/specviz2d/specviz2d.ipynb

```diff
@@ -0,0 +1,39 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# PREFIX\n",
+    "from jdaviz import Specviz2d\n",
+    "\n",
+    "specviz = Specviz2d(verbosity='JDAVIZ_VERBOSITY', history_verbosity='JDAVIZ_HISTORY_VERBOSITY')\n",
+    "specviz.load_data('DATA_FILENAME')\n",
+    "specviz.app"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.6-final"
```

Comment:
p.s. Tsk tsk, you still using Python 3.7? Astropy doesn't support it anymore, so you are not getting the latest astropy in that stack.